components/resources/quizzes/careerQuiz/career-quiz.tsx

```diff
@@ -34,132 +37,107 @@ const CareerQuiz = () => {
-          />
-        );
-      case Stage.EDUCATION:
-        return (
-          <EduBackground
-            onNextClick={handleNextClick}
-            onBackClick={handleBackClick}
-          />
-        );
-      case Stage.INDUSTRIES:
-        return (
-          <SkillSelections
-            onNextClick={handleNextClick}
-            onBackClick={handleBackClick}
-            title={"What industries are you interested in working?"}
-            body={"Select 1-3 choices."}
-            progress={35}
-            selections={[
-              "Advertising",
-              "Cybersecurity",
-              "Digital Media",
-              "Design",
-              "E-commerce",
-              "Entertainment",
-              "Fashion",
-              "Finance",
-              "Healthcare",
-              "Real Estate",
-              "Technology",
-              "Video Games",
-              "Science",
-              "Not Sure Yet",
-            ]}
-          />
-        );
-      case Stage.SKILLS:
-        return (
-          <SkillSelections
-            onNextClick={handleNextClick}
-            onBackClick={handleBackClick}
-            title={"What are your strongest skills?"}
-            body={"Select 1-3 choices."}
-            progress={60}
-            selections={[
-              "Writing code",
-              "Drawing",
-              "Design",
-              "Writing",
-              "Leading a team",
-              "Organization",
-              "Public speaking",
-              "Time Management",
-              "Math",
-              "Analyzing data",
-              "Critical thinking",
-              "Planning",
-            ]}
-          />
-        );
-      case Stage.TASKS:
-        return (
-          <SkillSelections
-            onNextClick={handleNextClick}
-            onBackClick={handleBackClick}
-            title={" What tasks would you prefer at work?"}
-            body={"Select 1-3 choices."}
-            progress={85}
-            selections={[
-              "Coordinate the launch of a product",
-              "Analyze social media campaigns",
-              "Find was to automate processes",
-              "Find trends in data",
-              "Study how people use apps",
-              "Build a company's brand",
-              "Lead a project from start to finish",
-              "Spot patterns in data",
-              "Write code to solve problems",
-            ]}
-          />
-        );
-      case Stage.BLUEPRINT:
-        return (
-          <BluePrint
-            onNextClick={handleNextClick}
-            onBackClick={handleBackClick}
-          />
-        );
-      case Stage.RESULTS:
-        saveUserPreferences({ variables: { objects: exampleUserPreferences } });
 
-        return <CareerResults onBackClick={handleBackClick} />;
-      default:
-        return null;
-    }
-  };
-
-  return <div>{renderStage()}</div>;
+  return <TrailAnimation open={open}>{items[stage]}</TrailAnimation>;
```

Comment:
the switch statement using stage is preferable to using an array of components. You should be able to use the switch statement inside here

Reply:
sounds good, went back to a switch statement. Out of curiosity, why are switch statements are preferable?